docs/rest_api.rst

```diff
@@ -532,10 +532,9 @@ Querying Events
 ===============
 
 Events are kept by the node. Once an event endpoint is queried the relevant events
-from either the beginning of time or the given block are returned. Events are returned in a sorted list with the most recent events on the top of the list.
+from either the beginning of time or the given block are returned. Events are returned in a sorted list with the most recent events on the top of the list. 
 
-Events are queried by two different endpoints depending on whether they are related
-to a specific channel or not.
+In Raiden we distinguish between two types of events: blockchain_events and raiden_events. blockchain_events are events that are being emitted by the smart contracts, as for raiden_events, those are events happening in the Raiden node for now, it's being used for debugging purpose. Be aware: All raiden events endpoints might get removed in the future. 
```

Comment:
Maybe format blockchain_events and raiden_events like this: `blockchain_events` and `raiden_events`